RibonDAO/app
# fix impact description and banner layout

# Description

- fix impact description and banner layout

## Type of change

Please delete options that are not relevant.


## How to test

- Verify the banner about integration in causes page when has a integration 
- Verify impact description from cta
